schema/mysql.sql

```diff
@@ -46,7 +46,8 @@ CREATE TABLE director_activity_log (
   INDEX search_idx (object_name),
   INDEX search_idx2 (object_type(32), object_name(64), change_time),
   INDEX search_author (author),
-  INDEX checksum (checksum)
+  INDEX checksum (checksum),
+  UNIQUE INDEX idx_checksum (checksum)
```

Comment:
```suggestion
  UNIQUE INDEX idx_checksum (checksum)
```
Setting the index once should suffice.